src/main/java/org/opentripplanner/model/TimetableSnapshot.java

```diff
@@ -111,38 +110,6 @@ public Timetable resolve(TripPattern pattern, LocalDate serviceDate) {
     return pattern.getScheduledTimetable();
   }
 
-  public void removeRealtimeUpdatedTripTimes(
```

Comment:
Other than changing the access from public to private it appears that this function has just been moved to a different location in the same file, but this is difficult to determine from the Github diff. Can you confirm that this method is just moved with no intended changes?

Unfortunately moving code around (as well as whitespace changes) can complicate a review because it's very hard to judge where the actual changes are.

Reply:
Yes I changed this to be private as this wasn't used from outside of this file anymore and since we have the private methods at the end of the files, I had to move this as well. I know this unfortunately messes up the diff sometimes.